### PR TITLE
[ADD] Default helpdesk tickets priority for specific customer

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**

### CODE REVIEW

File: ao_crm_helpdesk/views/res_partner_view.xml [hunk 1]
```diff
@@ -15,6 +15,13 @@
                     <field name="helpdesk_count" widget="statinfo" string="Helpdesk"/>
                 </button>
             </div>
+            <page name='sales_purchases' position="after">
+                <page name='helpdesk' string="Helpdesk">
+                    <group>
+                        <field name="helpdesk_default_priority"/>
```
Comment (LoisRForgeFlow) [inline ao_crm_helpdesk/views/res_partner_view.xml:21]: add `widget="priority"`

File: ao_crm_helpdesk/models/crm_helpdesk.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    def _get_default_priority(self):
+        if self.partner_id:
+            if self.partner_id.helpdesk_default_priority:
+                return self.partner_id.helpdesk_default_priority
+            else:
+                commercial_partner = self.partner_id.commercial_partner_id
+                if commercial_partner and \
+                        commercial_partner.helpdesk_default_priority:
+                    return commercial_partner.helpdesk_default_priority
+        return '1'
```
Comment (LoisRForgeFlow) [inline ao_crm_helpdesk/models/crm_helpdesk.py:21]: maybe copy this logic to an onchange as well?

File: ao_crm_helpdesk/models/crm_helpdesk.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
```
Comment (LoisRForgeFlow) [inline ao_crm_helpdesk/models/crm_helpdesk.py]: nitpicking, but 2019

**Review by LoisRForgeFlow — COMMENTED** [review]
Can you add a small test case?

File: ao_crm_helpdesk/models/crm_helpdesk.py (hunk 2 above)
Comment (LoisRForgeFlow) [reply ao_crm_helpdesk/models/crm_helpdesk.py:21]: nevermind you already have it

File: ao_crm_helpdesk/views/res_partner_view.xml (hunk 1 above)
Comment (LoisRForgeFlow) [reply ao_crm_helpdesk/views/res_partner_view.xml:21]: forget this too :laughing:  can be misleading as you do not have a way to distinguish between not set and 0 with the widget.

File: ao_crm_helpdesk/models/crm_helpdesk.py (hunk 3 above)
Comment (AdriaGForgeFlow) [reply ao_crm_helpdesk/models/crm_helpdesk.py]: Done :+1:

**Comment by AdriaGForgeFlow** [comment]
> Can you add a small test case?

Done :+1:

**Comment by AdriaGForgeFlow** [comment]
@lreficent Can you check the new commit in their test environment. Tests run OK on local.

**Pull request merged**